Apps/WebClient/src/ClientApp/src/models/userProfile.ts

```diff
@@ -15,6 +15,9 @@ export default interface UserProfile {
     // Indicates whether the email was verified.
     isEmailVerified: boolean;
 
+    // The email from the user's external identity provider.
+    identityProviderEmail?: string | null;
```

Comment:
The question mark at the end implies it could be undefined. I am guessing is only going to be a string or null right?

Reply:
I think this is the model that's also used when we save changes on the profile page.  In that case, we don't specify a value for the field.

Reply:
Added some more questions related to this field. It does not look correct to have two types of emails on the profile object

Reply:
We should discuss.

Reply:
Will swap to retrieving the email from the JWT on the front-end for a cleaner implementation.